ajax/checkDowntime.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * -------------------------------------------------------------------------
+ * Centreon plugin for GLPI
+ * -------------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of Centreon.
+ *
+ * Centreon is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Centreon is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Centreon. If not, see <http://www.gnu.org/licenses/>.
+ * -------------------------------------------------------------------------
+ * @copyright Copyright (C) 2022-2023 by Centreon plugin team.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @link      https://github.com/pluginsGLPI/centreon
+ * -------------------------------------------------------------------------
+ */
+
+use GlpiPlugin\Centreon\Host;
+
+Session::checkRight('computer', UPDATE);
+
+header('Content-Type: application/json');
+
+if (!isset($_GET['host_id'])) {
+    echo json_encode(['error' => 'Missing host_id']);
+    exit;
```

Comment:
No exit

https://glpi-developer-documentation.readthedocs.io/en/master/upgradeguides/glpi-11.0.html#handling-of-response-codes-and-early-script-exit